Source/Controller/DirectoryViewController.swift

```diff
@@ -345,12 +345,12 @@ extension DirectoryViewController: UITableViewDelegate {
         switch section {
         case .communityPubs:
             let star = communityPubs[indexPath.row]
-            let view = IdentityViewBuilder().build(identity: star.feed, botRepository: .shared, appController: .shared)
+            let view = IdentityViewBuilder.build(identity: star.feed)
```

Comment:
Should `IdentityViewBuilder.build(identity)` just return a `UIHostingController` instead of `some View`? It might save us a few more duplicated lines.

Reply:
👌